### PR TITLE
Review Code. 1

### PR DESCRIPTION
**Pull request by Rhotimee**
Kindly review and give feedback. Thank You.

### CODE REVIEW

File: server/config/config.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+module.exports = {
+  development: {
+    username: process.env.DB_USERNAME,
+    password: process.env.DB_PASSWORD,
+    database: process.env.DB_DEVELOPMENT,
+    host: process.env.HOST,
+    dialect: 'postgres',
+  },
+  test: {
+    username: process.env.DB_USERNAME,
```
Comment (seunzone) [inline server/config/config.js]: Great commit messages and I love the fact that you use environmental variable, good example of best practises

File: server/routes/v1/index.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+// Middleware
+import Middleware from '../../middlewares';
+
+const router = express.Router();
+
+router.get('/', (request, response) => {
+  response.redirect('/api-docs');
+});
+// POST register business
+router.post('/businesses', Middleware.isLoggedIn, Middleware.businessImageUpload, BusinessController.register);
```
Comment (seunzone) [inline server/routes/v1/index.js]: Nice, I see you have quite a number of middleware for  security sake, you could break these guys into mew lines to make your code more readable

File: client/components/AddBusiness/AddBusinessForm.jsx [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+import alertify from 'alertifyjs';
+
+
+/**
+ * @class AddBusinessForm
+ *
+ * @classdesc Add Business form
+ *
+ */
+class AddBusinessForm extends Component {
```
Comment (clintfidel) [inline client/components/AddBusiness/AddBusinessForm.jsx]: I like the fact that you paying attention to naming convention for classes, this is very commendable. I would however want you to take extra effort in incoporating same thing in necessary places.

File: client/components/BusinessDetails/BusinessDetails.jsx [hunk 4]
```diff
@@ -0,0 +1,267 @@
...
+   * @returns {void}
+   */
+  onSubmit(event) {
+    event.preventDefault();
+    // this.setState({ errors: {}, isLoading: true });
+    this.props.addReview(this.props.match.params.id, this.state).then(() => {
+      this.props.fetchReviews(this.props.match.params.id);
+      alertify.set('notifier', 'position', 'top-right');
+      alertify.success('Reviews Added');
+    });
```
Comment (clintfidel) [inline client/components/BusinessDetails/BusinessDetails.jsx]: nice job! take out commented code.

File: server/controller/reviewController.js [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+      }
+      Review.findAll({
+        where: { businessId: business.id },
+        include: [{
+          model: User,
+          as: 'reviewer',
+          attributes: ['firstName']
+        }]
+      }).then(reviews =>
+        // if (reviews.length === 0) {
```
Comment (ddouglasz) [inline server/controller/reviewController.js]: I understand that sometimes we like to comment out codes for better days, but please clean this up for professional purpose.

File: client/actions/businessAction.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+}
+
+export function oneBusiness(business) {
+  return {
+    type: 'FETCH_ONE_BUSINESS',
+    payload: business
+  };
+}
+
+export const setSearch = ({ search, type }) => (dispatch) => {
```
Comment (clintfidel) [inline client/actions/businessAction.js]: weldone declaring this function, however I don't seem to understand what this function does as the name is not descriptive, please try to improve on your naming convention.

File: client/components/AddBusiness/AddBusinessForm.jsx (hunk 3 above)
Comment (Rhotimee) [reply client/components/AddBusiness/AddBusinessForm.jsx]: @clintfidel, where are the necessary places I need to incorporate the naming convention?

**Pull request closed without merging**